lib/rucio/rse/rsemanager.py

```diff
@@ -703,6 +703,26 @@ def find_matching_scheme(rse_settings_dest, rse_settings_src, operation_src, ope
     raise exception.RSEProtocolNotSupported('No protocol for provided settings found : %s.' % str(rse_settings_dest))
 
 
+def _retry_protocol_stat(protocol, pfn):
+    """
+    try to stat file, on fail try again 1s, 2s, 4s, 8s, 16s, 32s later. Fail is all ail
+
+    :param protocol     The protocol to use to reach this file
+    :param pfn          Physical file name of the target for the protocol stat
+    """
+    from time import sleep
+    for attempt in range(6):
```

Comment:
In the baseclient there is a variable called `REQUEST_RETRIES`, I wonder if we should use this here as well.

Reply:
Looking through this it is specified in the Rucio configuration and used in the loop in line [298](https://github.com/rucio/rucio/blob/6d17a9dac8fb2844c10342dd74d39e9ff0f6161b/lib/rucio/client/baseclient.py#L298) of the baseclient. It sets a number of times to try opening a connection.

Using it here would overload the meaning: here the connection can be opened but it may take the storage element some time to recognise the data to be there. We could use this variable here, but should we be worried about introducing some interplay between what it does to connection retries and what it does in the RSE manager?